src/main/java/Dan/Task.java

```diff
@@ -0,0 +1,28 @@
+package Dan;
+
+public class Task {
+    protected String description;
+    protected boolean isDone = false;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return isDone ? "X" : " ";
+    }
+
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    public String toDataString(String separator) {
+        return String.format("%s%d%s%s", separator, (this.isDone) ? 1 : 0, separator, this.description);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[%s] %s",getStatusIcon(), this.description);
```

Comment:
There should be a space gap between , and getStatusIcon()

Reply:
Missed this out. Thank you!